spring-rabbit/src/test/java/org/springframework/amqp/rabbit/connection/LocalizedQueueConnectionFactoryIntegrationTests.java

```diff
@@ -63,6 +63,7 @@ public void testConnect() {
 		RabbitTemplate template = new RabbitTemplate(targetConnectionFactory);
 		template.convertAndSend("", queue.getName(), "foo");
 		assertEquals("foo", template.receiveAndConvert(queue.getName()));
+		admin.deleteQueue(queue.getName());
```

Comment:
Sorry - I was talking about something else.

Here, we never consume from the queue so it will not be deleted - auto-delete queues are deleted only when the last consumer disconnects.